services/fhir.ts

```diff
@@ -118,7 +114,7 @@ export function update(resource: AidboxResource, searchParams?: SearchParams): A
         return {
             method: 'PUT',
             url: `/${resource.resourceType}/${resource.id}`,
-            ...(versionId ? { headers: { 'If-Match': versionId } } : {}),
+            params: resource.id && versionId ? versionId : undefined,
```

Comment:
looks very strange. there are no params in this branch. Leave it as it was before `...(versionId ? { headers: { 'If-Match': versionId } } : {})`